src/android/StatusBar.java

```diff
@@ -227,20 +236,22 @@ private void setStatusBarBackgroundColor(final String colorPref) {
     }
 
     private void setStatusBarTransparent(final boolean transparent) {
-        if (Build.VERSION.SDK_INT >= 21) {
```

Comment:
Code indentation of the code inside this removed `if` should be adapted to match the added `this.transparent = transparent;`, right?

Reply:
Right. Fixes